### DISCUSSION TITLE
Hide the (repeated) header/title

### DISCUSSION THREAD

**Discussion by xihajun** [squidfunk/mkdocs-material#7145] [Q&A]
<img width="786" alt="image" src="https://github.com/squidfunk/mkdocs-material/assets/25631641/bf680e2f-1005-4cfa-88c0-852792586ab8">

Can anyone kindly point me out, how can we hide the duplicated header from the page?

**Comment by kamilkrzyskow** [comment]
Hello @xihajun,
are you using an H1 title tag at the top of your Markdown file? `# Advanced reasoning...`
I guess, you're not doing so, and using a smaller variant like H3 `###`, and therefore an H1 tag is automatically generated.
https://github.com/squidfunk/mkdocs-material/blob/5cb3117f50c78abfef6817b72cb09910decd272b/src/templates/partials/content.html#L31-L40
If you want to keep using the smaller variant, and prevent generation of the H1, you need to use customization and override the `content.html` with a version that doesn't have the "hack":
https://squidfunk.github.io/mkdocs-material/customization/#extending-the-theme
